src/main/java/bio/terra/cli/command/resources/create/BqDataset.java

```diff
@@ -18,12 +18,15 @@
 public class BqDataset extends BaseCommand {
   @CommandLine.Mixin ControlledResourceCreation controlledResourceCreationOptions;
 
-  @CommandLine.Option(names = "--dataset-id", required = true, description = "Big Query dataset id")
+  @CommandLine.Option(
+      names = "--dataset-id",
+      required = true,
+      description = "Big Query dataset id.")
```

Comment:
`BigQuery`

Reply:
I think I've made that mistake throughout the codebase. Will do a search and replace. Thanks for the catch.